packages/features/ee/round-robin/roundRobinManualReassignment.ts

```diff
@@ -0,0 +1,433 @@
+
+    let bookingLocation = booking.location;
+    if (eventType.locations.includes({ type: OrganizerDefaultConferencingAppType })) {
+      const newUserMetadataSafeParse = userMetadataSchema.safeParse(newUser.metadata);
+      const defaultLocationUrl = newUserMetadataSafeParse.success
+        ? newUserMetadataSafeParse?.data?.defaultConferencingApp?.appLink
+        : undefined;
+      const currentBookingLocation = booking.location || "integrations:daily";
+      bookingLocation =
+        defaultLocationUrl ||
+        getLocationValueForDB(currentBookingLocation, eventType.locations).bookingLocation;
+    }
+
+    const eventNameObject = {
+      attendeeName: responses?.name || "Nameless",
+      eventType: eventType.title,
+      eventName: eventType.eventName,
+      teamName: eventType.team?.name,
+      host: newUser.name || "Nameless",
+      location: bookingLocation || "integrations:daily",
+      bookingFields: { ...responses },
+      eventDuration: eventType.length,
+      t: newUserT,
+    };
+
+    const newBookingTitle = getEventName(eventNameObject);
+
+    booking = await prisma.booking.update({
+      where: { id: bookingId },
+      data: {
+        userId: newUserId,
+        title: newBookingTitle,
+      },
+      select: bookingSelect,
+    });
+  } else if (currentRRHost) {
+    // Update the round-robin host attendee
+    await prisma.attendee.update({
+      where: { id: currentRRHost.id },
+      data: {
+        name: newUser.name || "",
+        email: newUser.email,
+        timeZone: newUser.timeZone,
+        locale: newUser.locale,
+      },
+    });
+  }
+
+  const destinationCalendar = await getDestinationCalendar({
+    eventType,
+    booking,
+    newUserId,
+    hasOrganizerChanged,
+  });
+
+  const teamMembers = await getTeamMembers({
+    eventTypeHosts,
+    attendees: booking.attendees,
+    organizer: newUser,
+    previousHost: originalOrganizer,
+    reassignedHost: newUser,
+  });
+
+  const attendeePromises = booking.attendees.map(async (attendee) => ({
+    email: attendee.email,
+    name: attendee.name,
+    timeZone: attendee.timeZone,
+    language: {
+      translate: await getTranslation(attendee.locale ?? "en", "common"),
+      locale: attendee.locale ?? "en",
+    },
+    phoneNumber: attendee.phoneNumber || undefined,
+  }));
+
+  const attendeeList = await Promise.all(attendeePromises);
+
+  const evt: CalendarEvent = {
+    type: eventType.slug,
+    title: booking.title,
+    description: eventType.description,
+    startTime: dayjs(booking.startTime).utc().format(),
+    endTime: dayjs(booking.endTime).utc().format(),
+    organizer: {
+      email: newUser.email,
+      name: newUser.name || "",
+      timeZone: newUser.timeZone,
+      language: { translate: newUserT, locale: newUser.locale || "en" },
+    },
+    attendees: attendeeList,
+    uid: booking.uid,
+    destinationCalendar,
+    team: {
+      members: teamMembers,
```

Comment:
I am not sure for what `teamMembers` and `attendeeList` is needed, but why is the logic different than in `roundRobinAssignment`?

Reply:
The who section of the updated calendar event is also wrong, this is probably related to team members and `attendeeList`
<img width="367" alt="Screenshot 2024-10-21 at 2 42 19 PM" src="https://github.com/user-attachments/assets/8e5f93b1-86c1-4b48-ab81-27dd530f7fda">